autoload/pandoc/toc.vim

```diff
@@ -93,7 +93,8 @@ function! pandoc#toc#ReDisplay(bufname) abort
             endif
             let d.text = '· '.d.text
         endif
-        call setline(l, repeat(' ', 2*l:level-1). d.text)
+        let l:shift = exists('g:pandoc#toc#shift') ? g:pandoc#toc#shift : 2
```

Comment:
It's better to set the default in the `Init()` function and use it unconditionally here (there will always be a value)